WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift

```diff
@@ -111,6 +111,7 @@ private extension TopBannerView {
         } else {
             // It is necessary to remove the subview when no text, otherwise the stack view spacing stays.
             infoLabel.removeFromSuperview()
+            labelHolderStackView.removeFromSuperview()
```

Comment:
If `infoLabel` is added to  `labelHolderStackView`, is it necessary to keep `infoLabel.removeFromSuperview()`, or just removing `labelHolderStackView` will do the job?

Reply:
That's a good point, when this happens, keeping the `infoLabel` doesn't have any difference indeed. Removing the StackView holder does the job. Changed on https://github.com/woocommerce/woocommerce-ios/pull/8748/commits/eeffbfff1498bc59c3e4107fc54400f189fc5415